corehq/ex-submodules/toggle/models.py

```diff
@@ -45,6 +45,8 @@ def add(self, item):
         """
         Adds an item to the toggle. Only saves if necessary.
         """
+        from toggle.shortcuts import toggle_enabled
+        toggle_enabled.reset_cache((self.slug, item))
```

Comment:
should be in `bust_cache` with the other cache clear

Also since the memoized function is not a method on an object you don't need to pass in any args to clear the cache.